v1.0/install-cockroachdb.md

```diff
@@ -316,7 +318,10 @@ $(document).ready(function(){
 
   <ol>
     <li>
-      <p>Download the <a href="https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.linux-amd64.tgz" data-eventcategory="linux-binary-step1">CockroachDB {{ page.release_info.version }} archive for Linux</a>.</p>
+      <p>Download the CockroachDB archive for Linux:</p>
+
+      <div class="highlighter-rouge"><pre class="highlight"><code data-eventcategory="linux-binary-step2"><span class="gp" data-eventcategory="linux-binary-step2">$ </span>wget https://binaries.cockroachdb.com/cockroach-{{page.release_info.version}}.linux-amd64.tgz</code></pre>
```

Comment:
I think we still want a link somewhere. Either the URL itself could be a link (if that doesn't interfere with copy/paste), or we could make "CockroachDB archive" a link.